_docs/110_faq.md

```diff
@@ -112,19 +112,20 @@ you when you clone your repository.
 
 ### Should I `yadm add` encrypted files to repository?
 
-No, you should not. Files you want encrypted should be added to the file
-`.config/yadm/files.gpg` using the `yadm encrypt` command. Then
-`.config/yadm/files.gpg` should be added to the yadm repository. This way, only
-an encrypted collection of those files are put into the repository. After
-cloning or updating your repository, you can use `yadm decrypt` to extract those
-files from `.config/yadm/files.gpg`. See the
+No, you should not. Files you want encrypted should be added to the text file
```

Comment:
I think instead of "Files you want encrypted should be added ...", it might be clearer as "Names of files you want encrypted should be added ...". I'm trying to draw this distinction between names of files and their contents. Maybe I'm over-thinking it.

Reply:
Agreed. It's best to be as clear possible.